source/js/nav_jquery.js

```diff
@@ -1,8 +1,54 @@
 // Executes functions as soon as the page is fully loaded.
-$(document).ready(function() {
+$(document).ready(function () {
+    var isDesktop = DesktopCheck();
+    var isOverflow = OverflowCheck();
+
+    // This is how variable assignment is done in JavaScript.
+    let langBtn = "#langBtn";
+
+    // Set up variables for search.
+    let searchBtn = "#search";
+    let searchInput = ".search-input";
+    let searching = false;
+
+    // Set up variables for logo and title.
+    let logo = ".nav-phone .logo";
+    let title = ".nav-phone .title";
+
+
+    $(window).resize(function () {
+        isDesktop = DesktopCheck();
+        isOverflow = OverflowCheck();
+        if ($(window).width() == 670) {
+            if (searching) {
+                slideOut();
+                searching = false;
+            }
+        }
+    });
 
     // Toggles navigation menu on click of the navigation menu button.
-    $("#nav-btn").click(function() {
+    $("#nav-btn").click(function () {
+        openMenu(isDesktop);
+    });
+
+    function openMenu(isDesktop) {
+        if (isDesktop)
+            openDesktopMenu();
```

Comment:
here is a question of taste: in many teams people would either put the command on the same line as the `if` or `else` OR people would add the `{` `}`. what do you think?

Reply:
The discussion has moved to https://github.com/GalileoSchool/website/pull/7#discussion_r370421965